### DISCUSSION TITLE
Why does EarnApp limit the number of devices to 10?

### DISCUSSION THREAD

**Discussion by mrjkjxyz** [engageub/InternetIncome#186] [Q&A]
Why does EarnApp limit the number of devices to 10? @engageub 
![微信图片_20241218112221](https://github.com/user-attachments/assets/62cd2975-7d0b-461e-8367-506beb8c97f6)

**Comment by engageub** [comment]
Hello,
This has already been answered in the last comment of https://github.com/engageub/InternetIncome/discussions/156
 
 
![image](https://github.com/user-attachments/assets/c1fa0535-12c4-4b19-9a14-797d58546d47)


Thank you
